### DISCUSSION TITLE
jvm/stack-frame

### DISCUSSION THREAD

**Discussion by giscus** [itwanger/tobebetterjavaer-giscus#298] [Announcements]
# jvm/stack-frame

栈帧是运行时数据区中用于支持虚拟机进行方法调用和方法执行的数据结构。每一个方法从调用开始到执行完成，都对应着一个栈帧在虚拟机栈/本地方法栈里从入栈到出栈的过程。

https://javabetter.cn/jvm/stack-frame.html

**Comment by Yolo-XYX** [comment]
讲的很好尤其动态链接那部分，网上讲的都不太行
